frontend/components/User.tsx

```diff
@@ -8,7 +10,9 @@ export default function User(props: IUser) {
   return (
     <div>
       {props.role && roleMap.get(props.role)}
-      {props.nick}
+      <span className={`nick-${Math.abs(userHash(props.nick) % 16)}`}>
```

Comment:
Instead of using `Math.abs` around every single `userHash` call (I've seen three so far), maybe include it in the function, or at least add a wrapper function that does this for you?

Reply:
Maybe include the `%16` in the wrapper function as well.

Reply:
put into function